### PR TITLE
Optional connectionTimeout property support added

### PR DESCRIPTION
**Pull request by mP1**
- Closes https://issues.redhat.com/browse/WFARQ-3?_sscc=t

### CODE REVIEW

File: common/src/main/java/org/jboss/as/arquillian/container/CommonDeployableContainer.java [hunk 1]
```diff
@@ -103,6 +103,12 @@ public final void start() throws LifecycleException {
                 .setPort(containerConfig.getManagementPort())
                 .setAuthenticationConfigUri(authenticationConfig);
 
+        // only "copy" the timeout if one was set.
+        final int connectionTimeout = containerConfig.getConnectionTimeout();
+        if(0 != connectionTimeout) {
```
[the remark this answers is not in the document]
Comment (mP1) [reply common/src/main/java/org/jboss/as/arquillian/container/CommonDeployableContainer.java]: Changed to the gt test.

**Comment by mP1** [comment]
@jamezp Any chance for a new release ?

**Comment by jamezp** [comment]
@mP1 it's going to be at least a couple weeks before I'd be able to cut a release of this. It would be a beta release as well. Will that be an issue?

**Comment by mP1** [comment]
@jamezp No just wondering, thanks for the reply

**Comment by QazerLab** [comment]
@jamezp , is there a chance that this feature could be cherry-picked into and released within the `2.x` release line? Alternatively, is there any ETA for `3.0.0` stabilization?

**Comment by jamezp** [comment]
@QazerLab Since it's more of a feature I'd rather not back port it to 2.x. For 3.0.0 I'd say it's pretty stable now. Any additional changes would be minor bug fixes and changes/fixes for the bootable JAR container. We're likely go final by mid October though.

**Comment by QazerLab** [comment]
@jamezp Thanks for clarification! I'll be waiting for 3.0.0 then.

**Pull request merged**